app/src/main/java/com/yeonae/chamelezone/view/place/PlaceReviewTabFragment.kt

```diff
@@ -30,18 +40,32 @@ class PlaceReviewTabFragment : Fragment() {
             val intent = Intent(context, ReviewCreateActivity::class.java)
             startActivity(intent)
         }
-      
+
         setAdapter()
 
-        placeReviewRvAdapter.setOnClickListener(object : PlaceReviewTabRvAdapter.OnClickListener {
-            override fun onClick(like: Like) {
+        recycler_place_review.addOnItemTouchListener(object : RecyclerView.OnItemTouchListener {
+
+            override fun onInterceptTouchEvent(rv: RecyclerView, e: MotionEvent): Boolean {
+                val intent = Intent(context, MyReviewDetailActivity::class.java)
+                context?.startActivity(intent)
```

Comment:
```suggestion
                startActivity(intent)
```
fragment의 함수에도 있습니다!

Reply:
수정했습니당